data/mods/Aftershock/mobs/monster_faction.json

```
@@ -37,7 +37,7 @@
     "type": "MONSTER_FACTION",
     "name": "moxie",
     "base_faction": "mutant",
-    "neutral": "bio_machine",
+    "neutral": [ "bio_machine", "zombie" ],
     "by_mood": [ "small_animal", "herbivore" ],
     "hate": [ "insect", "human", "mutant", "WraitheonRobotics", "bot" ]
   },
```

```suggestion
  },
  {
  "type": "MONSTER_FACTION",
    "name": "zombie",
    "copy-from": "zombie",
    "neutral": [ "moxie" ]
  },
```
Probably needs some reformatting, and preferably would be located later in the file.

---

Thank you! I forget relationships aren't forcibly symmetric.